src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -9,7 +9,7 @@
  * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
  */
 public class NameContainsKeywordsPredicate implements Predicate<Person> {
-    private final List<String> keywords;
+    final List<String> keywords;
```

Comment:
Should this be `protected`? It is by default protected but you might want it to be private so it's better to be explicit